config/initializers/content_security_policy.rb

```diff
@@ -16,7 +16,16 @@
 
 #   # Specify URI for violation reports
 #   # policy.report_uri "/csp-violation-report-endpoint"
-# end
+
+  # from https://github.com/rails/webpacker#vue
+  Rails.application.config.content_security_policy do |policy|
```

Comment:
We're already in the security policy, so this do wrapper can be removed.

Reply:
I have no idea why, but removing this causes errors when loading charts on the run page related to this setting.

Reply:
I found the exact same behavior and it makes no sense, so we'll just leave it as is I guess.